File: backend/src/main/java/ca/metricalsky/winston/repository/CommentPropertiesRepository.java
package ca.metricalsky.winston.repository;

import ca.metricalsky.winston.entity.CommentPropertiesEntity;
import org.springframework.data.repository.ListCrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CommentPropertiesRepository extends ListCrudRepository<CommentPropertiesEntity, String> {

}

File: database/src/main/kotlin/ca/metricalsky/winston/database/repository/comment/CommentPropertiesRepository.kt
package ca.metricalsky.winston.database.repository.comment

import ca.metricalsky.winston.database.entity.comment.CommentPropertiesEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CommentPropertiesRepository: JpaRepository<CommentPropertiesEntity, String>
